index.html

```diff
@@ -19,7 +19,7 @@
     <div class="header">
```

Comment:
If you use HTML semantic elements it will ensure the coding standard.

Reply:
Hello @tarak-psdu 👋 , great usage of the inline comment! As requested by the previous reviewer, try to as be clear and polite as possible with your reviews. In this case, which semantic HTML elements should the student use? Don't be afraid of using emojis and words like 'kindly' and 'please'. This will make your reviews feel more like a suggestion instead of an order 😉.

Reply:
Please try to use HTML semantic elements. I mentioned it in the screenshot.
![Screenshot_5](https://user-images.githubusercontent.com/96407868/201276885-c0130bc6-2df1-4186-99b4-67bcfd1a9eb9.jpg)